sources/packages/web/src/components/common/students/applicationDetails/StudentAppealRequestsApproval.vue

```diff
@@ -1,9 +1,34 @@
 <template>
-  <body-header title="Student change">
+  <body-header title="Student change requests">
     <template #status-chip>
       <status-chip-requested-assessment :status="appealStatus" />
     </template>
   </body-header>
+  <div>
+    <p>
+      <strong>Instructions:</strong>
+      <br />
+    </p>
+    <ul>
+      <li>
+        View the change request and any supporting documentation on the student
+        application
+      </li>
+      <li>
+        Review the history of Request a Change submissions prior to approving
+        each new one to ensure continuity
+      </li>
+      <li>
+        Review all fields to ensure that information is consistent with the
+        students current circumstances
+      </li>
+      <li>
+        When the review is complete, come back to this page to approve or deny
+        the request
+      </li>
+    </ul>
+    <p></p>
+  </div>
```

Comment:
This whole section must be going into the subtitle slot of the `body-header`. Not outside it.